src/clj/test/cards/assets.clj

```diff
@@ -671,31 +742,65 @@
     (is (find-card "Braintrust" (:scored (get-corp))) "Braintrust is scored")))
 
 (deftest political-dealings
+  "Political Dealings - Full test"
```

Comment:
I agree with Neal that we should make these test comments use `;;` instead of strings as `deftest` doesn't actually support `doc-string`s (see comment [here](https://github.com/mtgred/netrunner/pull/2002#discussion_r80395634)).